lib/msf/core/rpc/v10/rpc_job.rb

```diff
@@ -19,14 +19,35 @@ def rpc_list
     res
   end
 
-  # Stops a job.
+  # Stops a job - alias for killing a session in `msfconsole`
```

Comment:
Looks like these comment changes are in the wrong place; This method is responsible for killing jobs, not sessions 👀

Reply:
Ah, you're right. My bad, I'll get that moved.

Reply:
I'll likely circle back and add a similar comment here once I get the job tracking updated 👍